src/vs/platform/extensionManagement/common/extensionManagement.ts

```diff
@@ -141,6 +141,8 @@ export interface IGalleryExtensionProperties {
 	dependencies?: string[];
 	extensionPack?: string[];
 	engine?: string;
```

Comment:
does renaming these `vscodeEngine` and `azdataEngine` make more sense for readability?

Reply:
I can rename it to azdataEngine, but we should keep the engine for vscode. as I mentioned in the PR description, we need to make sure the vscode extensions we would like to bring over to ADS also can work without modifications.

Reply:
I mean to make sure vscode extensions work with ads all we have to do is not change the property we read from package.json. We can also change the interfaces and what we call properties we our code since that won't affect what we read from the extensions.